vars/deploy.groovy

```diff
@@ -1,14 +1,21 @@
 #!/usr/bin/env groovy
 
-def call(String microservice, String aws_profile, String tag = null, boolean tagAfterDeployment = false, boolean run_tests = true) {
+def call(String microservice, String aws_profile, String tag = null, boolean tagAfterDeployment = false, boolean run_tests = true, smoke_tags = null) {
     tag = tag ?: gitCommit()
-    
+
+    smoke_version = 'v1'
+    if (tag != null) {
+        smoke_version = 'v2'
+    }
```

Comment:
don't understand this logic? If the tag is not null run a different version of smoke tests?

Reply:
oops ..copy paste error. It should be `smoke_tags`